src/reanimated2/core.ts

```diff
@@ -142,6 +142,16 @@ function valueUnpacker(objectToUnpack: any): any {
       handleCache!.set(objectToUnpack, value);
     }
     return value;
+  } else if (category === 'RemoteFunction') {
+    const fun = () => {
+      throw new Error(`Tried to synchronously call a non-worklet function
```

Comment:
Previously, this error message also contained the name of the function that was called. Is it possible to paste the function name here, at least in some cases?

Reply:
We could do that but not sure if its worth adding as part of this PR. This is already an improvement and I think it'd be better to spend time on solving issue with stack traces which should ideally pinpoint the exact line where the error happened rather than only showing the function name